src/main/java/lotto/service/LottoService.java

```diff
@@ -5,47 +5,24 @@
 import lotto.view.InputView;
 import lotto.view.ResultView;
 
-import java.util.HashMap;
-import java.util.LinkedList;
-import java.util.List;
 import java.util.Map;
 
 public class LottoService {
     public static void play() {
-        int boughtMoney = InputView.inputBoughtMoney();
-        validateBoughtMoney(boughtMoney);
-        int lottoCount = LottoServiceCalculator.getLottoCount(boughtMoney);
-        ResultView.printLottoCount(lottoCount);
-        List<Lotto> lottoList = createLottoList(lottoCount);
-        ResultView.printLottoNumbers(lottoList);
+        int boughtMoney = new BoughtMoney(InputView.inputBoughtMoney()).getBoughtMoney();
+        int totalCount = LottoServiceCalculator.getLottoCount(boughtMoney);
+        ManualLottos manualLottos = new ManualLottos(new ManualLottosCount(totalCount, InputView.inputManualLottoCount()));
+        int autoLottoCount = totalCount - manualLottos.getManualLottosCount();
+        manualLottos.createManualLottos(InputView.inputManualLottoNumbers(manualLottos.getManualLottosCount()));
+        Lottos lottos = new Lottos(manualLottos);
+        lottos.createAutoLottos(new ShuffledLottoNumbersStrategy(), totalCount - manualLottos.getManualLottosCount());
+
+        ResultView.printLottoCount(manualLottos.getManualLottosCount(), autoLottoCount);
+        ResultView.printLottoNumbers(lottos.getLottos());
         WinningLottoNumbers winningLottoNumbers = new WinningLottoNumbers(StringUtils.separate(InputView.inputWinningNumber()),
                                                                                     new BonusNumber(InputView.inputBonusNumber()));
-        Map<LottoPrize, Integer> lottoResultMap = createLottoResult(lottoList, winningLottoNumbers);
+        Map<LottoPrize, Integer> lottoResultMap = lottos.createLottosResult(winningLottoNumbers);
         ResultView.printWinningStatistics(lottoResultMap);
         ResultView.printProfitRate(LottoServiceCalculator.calculateProfitRate(lottoResultMap, boughtMoney));
```

Comment:
`LottoServiceCalculator`를 한번 제거해보면 어떨까요?
서비스 영역에 로직을 담은 util 클래스가 존재하는 것은 수정할 필요가 있어보입니다.

`Map<LottoPrize, Integer> lottoResultMap;`도 `Lottos`가 인스턴스 변수로 가지고있어보면 어떨까요? 
지금은 `Service`계층에만 `lottoResultMap`이 존재하는 형태이고,  
이는 결국 Service가 변수를 가지고 로직을 진행하는 것으로 점차 변화할 가능성이 큽니다.🙏

위 두 말을 종합해보면 아래와 같이 코드를 바꿔볼 수 있을 것 같습니다.

```suggestion
        lottos.setWinningLottoNumbers(winningLottoNumbers);
        ResultView.printWinningStatistics(lottos.getWinningMap());
        ResultView.printProfitRate(lottos.getProfitRate(boughtMoney));
```

Reply:
다만 이렇게되면 `createLottosResult`가 다시 private 메소드로 숨어버릴 가능성이 있겠는데요... 음 그 점은 저도 고민이네요.

Reply:
`getWinningMap`와 조합해서 테스트를 해보는 정도가 방법일 수 있겠네요.